### DISCUSSION TITLE
Unraid Template

### DISCUSSION THREAD

**Discussion by Bovive** [axewater/sharewarez#41] [General]
In reference to #4 I wanted to mention that I was able to create my own template for Unraid. It requires running Postgres in a separate container, but that is available as a Community App. I am not familiar with the process of submitting a template as a Community App, but if anyone is interested, I would be happy to assist with setting up the template.

Here is my [XML file](https://github.com/Bovive/unraid-templates/blob/main/SharewareZ/sharewarez.xml) template.

**Comment by axewater** [comment]
some of the people reporting problems setting up the app were using various NAS setups,  Unraid being one of them. My friend with an older Qnap also had issues.
NAS users are our audience, so we should cater to them and supply install guidance on all platforms.
but .. first lets make this thing worth sharing :)

**Comment by Bovive** [comment]
I cannot speak for the other platforms, but this Unraid template works and I'll maintain it until someone created a Community App for Unraid.
